src/utils/dif_upload.rs

```diff
@@ -77,6 +78,8 @@ impl<'a> Iterator for DifChunks<'a> {
 /// [`DifMatch`] instead of this instead.
 enum ParsedDif<'a> {
     Object(Box<Object<'a>>),
+    // TODO: consider making this an `Object` in symbolic-debuginfo?
```

Comment:
I don't think that's a good idea. PortablePdb has almost none of the features one would expect from an `Object`.